python/tvm/relax/transform/op_legalizer.py

```diff
@@ -340,6 +352,20 @@ def _nn_adaptive_max_pool2d(bb: BlockBuilder, args: List[Expr], attrs: Attrs, ou
     )
 
 
+def _cross_entropy(x, y):
+    return -topi.sum(topi.log(x) * y)
+
+
+def _nn_cross_entropy(bb: BlockBuilder, args: List[Expr], attrs: Attrs, output_shape: Expr):
+    return bb.call_te(cross_entropy, args[0], args[1])
+
+
+def _nn_softmax_cross_entropy(bb: BlockBuilder, args: List[Expr], attrs: Attrs, output_shape: Expr):
+    def softmax_cross_entropy(x, y):
+        return cross_entropy(topi.nn.softmax(x), y)
```

Comment:
:eyes:
```suggestion
        return _cross_entropy(topi.nn.softmax(x), y)
```

Reply:
Ahh I start to rebase and forget to change this.
But now I use the `te_` prefix to name the internal functions (see the lastest commit). How about this? 👀

Reply:
Yeah thanks for the change!